### ISSUE TITLE
Is the vita port dead or just hitting roadblocks?

### ISSUE THREAD

**Issue by NeonLightning** [Keriew/augustus#198]
well the title says it all. just curious why the vita port is behind.

**Comment by crudelios** [comment]
TL;DR: It's hard to mantain.

Long answer: First, the Vita port was not created by any of Augustus or Julius' developers, but by a Vita developer. The original port was hackish and required a lot of effort to mantain. This is especially noticeable on Augustus after implementing zoom support, which requires massive changes to the Vita and Switch ports.

On the past few months I bought a Vita and started getting involved in both Vita and Switch ports of Julius, having basically merged the codebase between all platforms.

That, on its hand, made merging the new Julius changes to Augustus more complicated. Expect easier compatibility in the future, when Augustus catches up with Julius.

Moreover, performance on Vita is poor. It's hard enough to get a stable framerate on Julius, let alone on Augustus which is more taxing.

There have been recent updates to VitaSDK that should improve performance a bit, but I'm still testing things out in Julius.

Then there's the issue of how to bundle the new custom assets to the Vita port. I'm thinking about how to do that for Vita, Switch, Android, Linux and macOS at the same time.

After all I've said is done, maybe there will be a Vita port. In the end it will depend on performance alone.

**Comment by NeonLightning** [comment]
that makes sense thanks for the detailed answer.
